cmd/cli/select_context.go

```diff
@@ -0,0 +1,56 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/ergomake/layerform/internal/lfconfig"
+)
+
+func init() {
+	configCmd.AddCommand(configSelectContextCmd)
+}
+
+var configSelectContextCmd = &cobra.Command{
+	Use:   "select-context <name>",
+	Short: "Select a context entry from layerform config file",
+	Long: `Select a context entry from layerform config file.
+	
+  Selecting a name that does not exist will return error.`,
+	Example: `# Select a context
+layerform config select-context local-example`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		name := args[0]
+
+		cfg, err := lfconfig.Load("")
+		if err != nil && !errors.Is(err, os.ErrNotExist) {
+			fmt.Fprintf(os.Stderr, "%s\n", errors.Wrap(err, "fail to open config file"))
+			os.Exit(1)
+		}
+
+		_, ok := cfg.Contexts[name]
+		if !ok {
+			fmt.Fprintf(
+				os.Stderr,
+				"context %s does not exist\n",
```

Comment:
Can we change this to:
```suggestion
				"no context exists with the name \"%s\".\n",
```

Reply:
updated https://github.com/ergomake/layerform/pull/96/commits/978e3c6bf38d9b22e48eaffe01c2c63f6d632b03